docker/oso-monitor-sso/src/Dockerfile.j2

```diff
@@ -35,10 +36,13 @@ ADD root/ /root/
 ADD ops-run-in-loop start.sh /usr/local/bin/
 
 # Create ops-runner.log file with proper permissions
-RUN touch /var/log/ops-runner.log && chmod 664 /var/log/ops-runner.log
-
 # Create monitor-sso.log file with proper permissions
-RUN touch /var/log/monitor-sso.log && chmod 664 /var/log/monitor-sso.log
+# Create symlinks to secrets
+RUN touch /var/log/ops-runner.log && chmod 664 /var/log/ops-runner.log && \
```

Comment:
docker recommends consolidating RUN layers when possible so that you don't end up with lots of extra layers. It also speeds up the builds. I added some steps to make symlinks from the /etc/openshift_tools/* config files to their locations as secrets.